functions/view/advertiser/demographics-survey.hbs

```diff
@@ -0,0 +1,179 @@
+                type='radio'
+                id='north-america'
+                name='continent'
+                value='north-america'
+              />&nbsp North America</label>
+            <label class='demo__radio-label' for='south-america'><input
+                type='radio'
+                id='south-america'
+                name='continent'
+                value='south-america'
+              />&nbsp South America</label>
+          </div>
+          <br />
+          <div>
+            <button class='demo__submit-button' type='submit'>Submit</button>
+            <button class='demo__continue-button'>
+              <a href='{{{publisherAUrl}}}/demographics-measurement'>
+                Continue
+              </a>
+            </button>
+          </div>
+        </form>
+      </card>
+      <card class='demo__description'>
+        <h4>Description</h4>
+        <p>
+          You may want to measure the demographics of the users who has seen some content you have embedded across
+          different sites.
+        </p>
+        <p>
+          For this demo, visit the survey page to enter your demographic data where the information you enter is added
+          to Shared Storage. Then visit either Publisher A or B sites which will trigger the aggregatable report to be
+          sent once across both sites. The content ID, age group ID, and the geography ID dimensions are encoded into
+          the aggregation key (bucket), and the count is used as the aggregatable value.
+        </p>
+        <p>
+          The summary report generated will provide information such as "Approximately 391 users who have seen the
+          content ID 743 are between the age of 18-25 and are from New York City."
+        </p>
+        <h4>Code</h4>
+        <ul>
+          <li>
+            <a
+              href='https://github.com/GoogleChromeLabs/shared-storage-demo/blob/main/sites/advertiser/private-aggregation/demographic-measurement.js'
+              target='_blank'
+            >Iframe logic</a>
+            (embedded into the publisher page)
+          </li>
+          <li>
+            <a
+              href='https://github.com/GoogleChromeLabs/shared-storage-demo/blob/main/sites/advertiser/private-aggregation/demographic-measurement-worklet.js'
+              target='_blank'
+            >Worklet</a>
+            (loaded and executed by the iframe logic)
+          </li>
+        </ul>
+      </card>
+    </div>
+    <card class='demo__instructions'>
+      <p>
+        Shared Storage API is not enabled in your browser. Follow the
+        <a
+          href='https://developer.chrome.com/docs/privacy-sandbox/shared-storage/#try-the-shared-storage-api'
+          target='_blank'
+        >instructions</a>, and enable the the
+        <b>Privacy Sandbox Ads APIs</b>
+        experiment flag at
+        <b>chrome://flags/#privacy-sandbox-ads-apis</b>
+        with Chrome 104.0.5086.0 and above to test this Shared Storage API demo.
```

Comment:
We may need to update this milestone (here and elsewhere when PA is needed). I think we added PA to the Privacy Sandbox Ads API flag on "107.0.5292.0"